include/dicom/dicom.h

```diff
@@ -2372,16 +2372,50 @@ void dcm_bot_destroy(DcmBOT *bot);
  */
 
 /**
- * Create a File.
+ * A set of IO functions, see dcm_file_create_io().
+ */
+typedef struct _DcmIO {
+    /** Open an IO object */
+    void *(*open)(DcmError **error, void *client);
+    /** Close an IO object */
+    int (*close)(DcmError **error, void *data);
+    /** Read from an IO object, semantics as POSIX read() */
+    int64_t (*read)(DcmError **error, void *data, char *buffer, int64_t length);
+    /** Seek an IO object, semantics as POSIX seek() */
+    int64_t (*seek)(DcmError **error, void *data, int64_t offset, int whence);
+} DcmIO;
+
+/**
+ * Create a File that reads using a set of DcmIO functions.
+ *
+ * :param io: Set of read functions for this DcmFile
```

Comment:
The `error` parameter is missing from the docstring (and from others)

Reply:
Ooops! Fixed, thanks.